samples/matter/window_covering/README.rst

```diff
@@ -38,6 +38,22 @@ The following movement modes are available:
 
 See `User interface`_ for information about how to switch the movement modes.
 
+Synchronized Sleepy End Device activity
+=======================================
+
+Because it is working as a Thread Synchronized Sleepy End Device, the window covering device based on this sample is synchronized with its parent router and uses the radio only at scheduled intervals.
+During those intervals, the device waits for the router to send it any data related to the window cover movement.
+The SSED does not actively communication to the router and goes into the idle mode in-between the scheduled activity periods.
+Overall, this further reduces energy consumption compared with a standard Thread SED.
```

Comment:
I think we could add additional information regarding synchronization maintenance, e.g. "The SSED device requires sending packets once for a time to keep synchronization with the router. In case of the absence of application related traffic for a longer period of time, the SSED sends data poll request packet to synchronize with the parent."